cmake/modules/FindIspc.cmake

```diff
@@ -6,17 +6,17 @@
 # set(myTarget_ISPC_OBJ "")
 # find_package(Ispc)
 # ispc_compile_single_file(myTarget_ISPC_OBJ SampleGen.ispc avx2) # avx2 instruction set support
-# add_executable(myTarget EXCLUDE_FROM_ALL ${myTarget_SRC} ${myTarget_INC} ${myTarget_ISPC_OBJ})
+# add_executable(myTarget ${myTarget_SRC} ${myTarget_INC} ${myTarget_ISPC_OBJ})
```

Comment:
note to self: was this reflected in jenkins file

Reply:
I will remove it from Jenkinsfile

Reply:
I meant, AdcSimulator and AdcFileStreamer should be removed as explicit from the jenkinsfile, but we allready have a comment on that :)